packages/channels/src/teams/README.md

```diff
@@ -0,0 +1,29 @@
+### Sending
+
+| Channels | Teams |
+| -------- | :---: |
+| Text     |  ✅   |
+| Image    |  ✅   |
+| Choice   |  ✅   |
+| Dropdown |  ✅   |
+| Card     |  ✅   |
+| Carousel |  ✅   |
+| File     |  ✅   |
+| Audio    |  ✅   |
+| Video    |  ✅   |
+| Location |  ✅   |
+
+### Receiving
+
+| Channels      | Teams |
+| ------------- | :---: |
+| Text          |  ✅   |
+| Quick Reply   |  ✅   |
+| Postback      |  ✅   |
+| Say Something |  ✅   |
```

Comment:
I sorted these out. Turns out this channel only received text, even if it was quick reply, postback and say something. Now all three work.